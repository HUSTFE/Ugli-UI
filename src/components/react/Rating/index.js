/* eslint-disable */
import React, { Component } from 'react'
import PropTypes from 'prop-types'

import styles from '@style/Rating/index.sass'

class Rating extends Component {
  render() {
    return <div>Rating</div>
  }
}

export default Rating
