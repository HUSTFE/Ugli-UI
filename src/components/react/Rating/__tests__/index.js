import React from 'react'
import renderer from 'react-test-renderer'
import Rating from '../index'

describe('Rating', () => {
  it('snake spinner render without crash', () => {
    const component = renderer.create(<Rating />)
    const json = component.toJSON()
    expect(json).toMatchSnapshot()
  })
})
