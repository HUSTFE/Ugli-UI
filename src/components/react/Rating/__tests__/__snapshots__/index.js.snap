// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Rating snake spinner render without crash 1`] = `
<div>
  Rating
</div>
`;
